# This file holds an array of type detected objects msg	
objectDetection[] detections
